lineage/figures/common.py

```diff
@@ -328,15 +328,10 @@ def commonAnalyze(list_of_populations, num_states, xtype="length", **kwargs):
     list_of_fpi = kwargs.get("list_of_fpi", [None] * len(list_of_populations))
     list_of_fT = kwargs.get("list_of_fT", [None] * len(list_of_populations))
     list_of_fE = kwargs.get("list_of_fE", [None] * len(list_of_populations))
-    if num_states == 2:
-        predicted_num_states = kwargs.get("predicted_num_states", 2)
-    elif num_states == 3:
```

Comment:
Was this here for some reason before? I can see getting rid of this code, but I can't understand why it was written this way in the first place...

Reply:
`predicted_num_states` has not been referenced anywhere else, so I think there was no point having it. I am not sure why this was written like this.